controllers/controllers.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"github.com/go-logr/logr"
+	"github.com/redhat-appstudio/release-service/controllers/release"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+// setupFunctions is a list of register functions to be invoked so all controllers are added to the Manager
+var setupFunctions = []func(manager.Manager, *logr.Logger) error{
+	release.SetupController,
+}
+
+// SetupControllers adds all controllers to the Manager.
```

Comment:
Might be nice to say `adds all controllers defined in setupFunctions to the Manager` but I am fine if you don't like that and want to leave it as is

Reply:
I wrote something similar:
```
// SetupControllers invoke all SetupController functions defined in setupFunctions, setting all controllers up and
// adding them to the Manager.
```